src/component/StyleComponent.js

```diff
@@ -0,0 +1,26 @@
+import styled, { css } from "styled-components";
+
+const Box = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  padding-bottom: 20px;
+  border-bottom: 1px solid gray;
+`;
+
+const List = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+  height: 200px;
+  overflow: hidden auto;
+
+  ${(props) =>
+    props.done &&
+    css`
+      text-decoration: line-through;
+      color: gray;
+    `}
```

Comment:
```suggestion
  ${({ done }) =>
    done &&
    css`
      text-decoration: line-through;
      color: gray;
    `}
```

속성에 따른 스타일을 적용해주셨네요👍 여기에 `destructuring`을 사용하면 더 깔끔할 것 같아요!